### ISSUE TITLE
Click warnings

### ISSUE THREAD

**Issue by cpecquerie** [Diaoul/subliminal#498]
Click python lib gives warnings at each subliminal 1.0.1 launch.
See bellow output of subliminal download --help  command.                                                                                                                                                               

```
/usr/local/lib/python2.7/dist-packages/subliminal/cli.py:115: Warning: Click detected the use of the unicode_literals __future__ import.  This is heavily discouraged because it can introduce subtle bugs in your code.  You should instead use explicit u"" literals for your unicode strings.  For more information see http://click.pocoo.org/python3/
  @click.pass_context
/usr/local/lib/python2.7/dist-packages/subliminal/cli.py:145: Warning: Click detected the use of the unicode_literals __future__ import.  This is heavily discouraged because it can introduce subtle bugs in your code.  You should instead use explicit u"" literals for your unicode strings.  For more information see http://click.pocoo.org/python3/
  @click.pass_context
/usr/local/lib/python2.7/dist-packages/subliminal/cli.py:172: Warning: Click detected the use of the unicode_literals __future__ import.  This is heavily discouraged because it can introduce subtle bugs in your code.  You should instead use explicit u"" literals for your unicode strings.  For more information see http://click.pocoo.org/python3/
  @click.pass_obj```

**Comment by Diaoul** [comment]
Please search closed issues #495

**Comment by cpecquerie** [comment]
oups sorry
